test/entry_point_func.ll

```diff
@@ -12,10 +12,47 @@ define spir_kernel void @testfunction() {
    ret void
 }
 
+define spir_kernel void @callerfunction() {
+   call spir_kernel void @testfunction()
+   call spir_kernel void @testdeclaration()
+   ret void
+}
+
+declare spir_kernel void @testdeclaration()
+
 ; Check there is an entrypoint and a function produced.
-; CHECK-SPIRV: EntryPoint 6 [[EP:[0-9]+]] "testfunction"
-; CHECK-SPIRV: Name [[FUNC:[0-9]+]] "testfunction"
-; CHECK-SPIRV: Decorate [[FUNC]] LinkageAttributes "testfunction" Export
-; CHECK-SPIRV: Function 2 [[FUNC]] 0 3
-; CHECK-SPIRV: Function 2 [[EP]] 0 3
-; CHECK-SPIRV: FunctionCall 2 8 [[FUNC]]
\ No newline at end of file
+; CHECK-SPIRV: EntryPoint 6 [[#CallerEn:]] "callerfunction"
+; CHECK-SPIRV: EntryPoint 6 [[#TestEn:]] "testfunction"
+; CHECK-SPIRV: Name [[#TestDecl:]] "testdeclaration"
+; CHECK-SPIRV: Name [[#TestFn:]] "testfunction"
```

Comment:
`having EntryPoint function and a function that shares the name via OpName might be confusing` - why do we have 2 checks for `testfunction` here? Am I missing something?

Reply:
Thanks for the clarification